### PR TITLE
Bug fixes for 4.5.0 picked from main

### PR DESCRIPTION
**Pull request by ThorodanBrom**
Cherry-picked 2 commits from main here
- https://github.com/datakaveri/iudx-aaa-server/commit/f6b38b57df4cd55ac456ca4e630b58da13eaa4b0
- https://github.com/datakaveri/iudx-aaa-server/commit/6b6ab52e01695225c4a72ccf8f17bacee0fe5552

### CODE REVIEW

**Comment by ThorodanBrom** [comment]
retest this please

**Comment by ThorodanBrom** [comment]
The ZAP/Integration test errors here are due to cat API changes, we can't do anything about them yet.

File: docker/runIntegTests.sh [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 
-files_present=$(ls ./src/main/resources/db/migration | wc -l)
-new_version=$(echo "$files_present+1" | bc)
-cp /home/ubuntu/configs/aaa-Add_Integration_Test_data.sql ./src/main/resources/db/migration/V${new_version}__Add_Integration_Test_data.sql
-mvn flyway:migrate -Dflyway.configFiles=/home/ubuntu/configs/aaa-flyway.conf
\ No newline at end of file
+cp ./src/test/resources/V1000__Add_Integration_Test_data.sql ./src/main/resources/db/migration/
+mvn flyway:migrate -Dflyway.configFiles=/home/ubuntu/configs/aaa-flyway.conf
```
Comment (karun-singh) [inline docker/runIntegTests.sh:4]:
```suggestion
mvn flyway:migrate -Dflyway.configFiles=/home/ubuntu/configs/4.5.0/aaa-flyway.conf
```

**Review by karun-singh — CHANGES_REQUESTED** [review]

**Review by karun-singh — APPROVED** [review]
LGTM

**Pull request merged**